templates/profile.html

```diff
@@ -7,98 +7,105 @@
 <h1>Create a profile</h1>
 {% endif %}
 
-<ul>
-  <li>Full name: <span data-edit-name=name data-edit-attrs="required">{{ user.profile.name or '' }}</span></li>
-  <li>Website: <span data-edit-name=website data-edit-attrs="type=url"><a href="{{ user.profile.website }}" target=_blank>{{ user.profile.website }}</a></span></li>
-</ul>
+<main>
```

Comment:
I suppose that main should be in the base layout and eventually the `<h1>` title inside of it.

Reply:
sure